wormchain/Makefile

```diff
@@ -78,3 +93,44 @@ bootstrap:
 clean:
 	rm -rf build/wormchaind build/wormchaind-* build/**/*.db build/**/*.wal vue
 	echo "{\"height\":\"0\",\"round\":0,\"step\":0}" > build/data/priv_validator_state.json
+
+#####################
+## INTERCHAINTESTS ##
+#####################
+
+# Generate Wormchain Image
+local-image: build/wormchaind
+	docker build -t wormchain:latest -f Dockerfile.ict ..
+
+# Individual Tests ($$ is interpreted as $)
+rm-testcache:
+	go clean -testcache
+
+ictest-cancel-upgrade: rm-testcache
+	cd interchaintest && go test -race -v -run ^TestCancelUpgrade$$ ./...
+
+ictest-chain-start: rm-testcache
```

Comment:
When running this test, it fails with:

```
wormchain:local: pull image wormchain:local: Error response from daemon: pull access denied for wormchain
```

It then only passed if I tagged the local docker image built for wormchain ICT as "local" instead of "latest", we may need to update the make command on line 102 above to build the image with the tag local instead of latest, like:

```
local-image: build/wormchaind
	docker build -t wormchain:local -f Dockerfile.ict ..
```

We may want to consider a way to ensure the local docker image is built with the latest version of the current branch's wormchaind software before running the tests.

Reply:
Good points there, I agree that rebuilding the image every test would be an unnecessary annoyance and make test runs take longer, especially if chain code stays the same. I will spend some time thinking about it, but this should work just fine for now.